examples/example_basicusage.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a3681317-4fd7-434a-9ac5-de994c216826",
+   "metadata": {},
+   "source": [
+    "# Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8dad5bd8-d71d-4b37-a513-88689c9bb504",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install Opti-HPLC-Handler # Inital installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97587f1d-4223-48ab-853e-793e93c5bdff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install Opti-HPLC-Handler --upgrade # Update"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef4c3950-5578-4d87-8c11-184092f12e9f",
+   "metadata": {},
+   "source": [
+    "# Importing and Login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0990b0e4-2fcd-4e60-ae65-c76a978fd74a",
+   "metadata": {},
+   "source": [
+    "Choose project and log in"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "c9673853-ff9b-464e-a498-ba74d0649fac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Please enter the password for user slcb:  ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from OptiHPLCHandler import EmpowerHandler\n",
+    "handler=EmpowerHandler(project=\"SLCB_2023_DEV\",address=\"http://epdkhqr01018.corp.novocorp.net:3076/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b90cac75-a886-4584-9eb9-6ca474b0845f",
+   "metadata": {},
+   "source": [
+    "# Connection Token "
```

Comment:
A description of what they token is useful for would be nice (e.g. "for use in swagger UI or for direct use of Empower Web API)